### DISCUSSION TITLE
Showcase your work in this thread!

### DISCUSSION THREAD

**Discussion by nbrown02** [nbrown02/ADO-Process-Behaviour-Chart#5] [Show and tell]
Modified the template for your own context and finding new uses? Share it here with instructions/links to your forked repo(s) !

**Comment by huserben** [comment]
## [Total-Work-Item-Age-Projection-Power-BI-Template](https://github.com/huserben/Total-Work-Item-Age-Projection-Power-BI-Template)
This Power BI Template is there to visualize the Process Behavior Charts (X-Charts) of Total Work Item Age (TWIA) as well as Work in Progress (WIP) together with the current TWIA and WIP and a projection about TWIA for the next few days.

All the elements in the report should aid you in answering the question: "Should we pull in new work?".

![Report](https://github.com/huserben/Total-Work-Item-Age-Projection-Power-BI-Template/assets/5486874/cc982cf9-de17-4cac-afe7-7db2ad850b9d)

**Reply by huserben** [reply]
@nbrown02 Finally added a new version of the template and added a proper ReadMe in the repository.

**Reply by nbrown02** [reply]
Love it thanks for sharing!
